File: solutions/python/roman-numerals/1/roman_numerals.py
def roman(number):
    

    roman_map = [
    (1000, "M"),
    (900, "CM"),
    (500, "D"),
    (400, "CD"),
    (100, "C"),
    (90, "XC"),
    (50, "L"),
    (40, "XL"),
    (10, "X"),
    (9, "IX"),
    (5, "V"),
    (4, "IV"),
    (1, "I"),
    ]

    if number <= 0 or number > 3999:
        raise ValueError("number out of range")

    result=''
    for value, symbol in roman_map:
        while number >= value:
            result += symbol
            number -= value
    
    return result

File: solutions/javascript/roman-numerals/1/roman-numerals.js
//
// This is only a SKELETON file for the 'Roman Numerals' exercise. It's been provided as a
// convenience to get you started writing code faster.
//
const valMap = [
  [1000, "M"],
  [900,  "CM"],
  [500,  "D"],
  [400,  "CD"],
  [100,  "C"],
  [90,   "XC"],
  [50,   "L"],
  [40,   "XL"],
  [10,   "X"],
  [9,    "IX"],
  [5,    "V"],
  [4,    "IV"],
  [1,    "I"]
];
export const toRoman = (number) => {
let result = ""
        for (let [value, symbol] of valMap){
            while( number >= value){
              
             
             result += symbol;
             number -= value;
        }
        }
        return result;


  
};
